### PR TITLE
Update README to include CodeGen

### LINKED ISSUE #665: Update README to include Fabulous.CodeGen

### PR DESCRIPTION
**Pull request by TimLariviere**
Closes #665

### CODE REVIEW

File: Fabulous.CodeGen/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Add support for 2 new modes, so it can be run by anyone to generate bindings on any given dll
+   * `generate-mapping`: Generate the JSON mapping file with all controls and properties
+   * `bind`: Generate the F# bindings code for a given mapping file
+
+* Docs
+  * Generate better `///` docs
+
+## Contributing
+
+Please contribute to this library through issue reports, pull requests, code reviews and discussion.
```
Comment (SergejDK) [inline Fabulous.CodeGen/README.md]: Shouldn't it be `discussions` ? 
Currently it is `discussion`.

File: README.md [hunk 2]
```diff
@@ -67,8 +76,8 @@ If you wish to see more support for Fabulous.StaticView, please consider contrib
 
 ## Contributing
 
-Please contribute to this library through issue reports, pull requests, code reviews and discussion.
+Please contribute to this repository through issue reports, pull requests, code reviews and discussion.
```
Comment (SergejDK) [inline README.md]: Same?

File: Fabulous.CodeGen/README.md (hunk 1 above)
Comment (TimLariviere) [reply Fabulous.CodeGen/README.md]: As far as I know, both are correct here. `discussion` can be uncountable for things like the general act of discussing. Though I'm not a native English speaker, so I could be wrong. :)

Comment (SergejDK) [reply Fabulous.CodeGen/README.md]: You are right in this sentence it is oaky to use one or the other. I am not a native english speaker too :D

**Pull request merged**